packages/cicero-core/test/metadata.js

```diff
@@ -178,7 +178,37 @@ describe('Metadata', () => {
                     ergo: '0.1.0',
                     language: 'ergo'
                 },
-            }, null, {})).should.throw('The template targets Cicero (0.0.0) but the Cicero version is ');
+            }, null, {})).should.throw('The template targets Cicero version 0.0.0 but the current Cicero version is ');
+        });
+
+        it('should get the displayName from packageJson', () => {
```

Comment:
Would it make sense to test inputting a `displayName` that is not already formatted? Such as `my-diSPLAY_name` or something similar.

Reply:
Yes, let me improve the coverage there.